http_service/bugbug_http/models.py

```diff
@@ -32,41 +30,34 @@
     "testlabelselect",
     "testgroupselect",
 ]
+
 DEFAULT_EXPIRATION_TTL = 7 * 24 * 3600  # A week
+redis = Redis.from_url(os.environ.get("REDIS_URL", "redis://localhost/0"))
 
 
-MODEL_LAST_LOADED: Dict[str, datetime] = {}
-MODEL_CACHE: Dict[str, Model] = {}
+def load_model_for_service(model_name):
+    LOGGER.info("Recreating the %r model in cache" % model_name)
+    try:
+        return load_model(model_name)
+    except FileNotFoundError:
+        if ALLOW_MISSING_MODELS:
+            LOGGER.info(
+                "Missing %r model, skipping because ALLOW_MISSING_MODELS is set"
+                % model_name
+            )
+            return None
+        else:
+            raise
 
 
-redis = Redis.from_url(os.environ.get("REDIS_URL", "redis://localhost/0"))
+MODEL_CACHE: ReadthroughTTLCache[str, Model] = ReadthroughTTLCache(
+    timedelta(hours=2), load_model_for_service
+)
+MODEL_CACHE.start_ttl_thread()
 
 
 def get_model(model_name):
-    if model_name not in MODEL_CACHE:
-        LOGGER.info("Recreating the %r model in cache" % model_name)
-        try:
-            model = load_model(model_name)
-        except FileNotFoundError:
-            if ALLOW_MISSING_MODELS:
-                LOGGER.info(
-                    "Missing %r model, skipping because ALLOW_MISSING_MODELS is set"
-                    % model_name
-                )
-                return None
-            else:
-                raise
-
-        # Cache the model only if it was last used less than one hour ago.
-        if model_name in MODEL_LAST_LOADED and MODEL_LAST_LOADED[
-            model_name
-        ] > datetime.now() - relativedelta(hours=1):
-            MODEL_CACHE[model_name] = model
-    else:
-        model = MODEL_CACHE[model_name]
-
-    MODEL_LAST_LOADED[model_name] = datetime.now()
-    return model
+    return MODEL_CACHE.get(model_name)
```

Comment:
Since this function is a one-liner, we can just remove it and use `MODEL_CACHE.get(model_name)` directly